ansible/roles/contiv/tasks/restart_netmaster.yml

```diff
@@ -0,0 +1,35 @@
+- name: stop netmaster
```

Comment:
restart_netmaster.yml performs the same tasks as net master.yml? Is this really needed? Would a handler and notify help? Take a look at the following for add'l info on the handler/notify idea:

roles/docker/handlers/main.yml